### PR TITLE
Docs: Add Onboarding Dashboard guide and Glossary

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR adds the Onboarding dashboard doc and the first version for the glossary.

Preview: 
- [Dashboard](https://faststore-site-git-docs-onboarding-dashboard-faststore.vercel.app/docs/getting-started/1-faststore-onboarding/dashboard)
- [Glossary](https://faststore-site-git-docs-onboarding-dashboard-faststore.vercel.app/docs/getting-started-glossary)

### CODE REVIEW

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 1]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+
+Check out the table below for more information:
+
+
+| Field                 | Description                                                                                                                           |
+| :-------------------- | :------------------------------------------------------------------------------------------------------------------------------------ |
+| **Preview FastStore** | Opens a preview of your store in a new tab.                                                                                           |
+| **Deployment**        | Opens the deployed version of the store in a new tab.                                                                                 |
+| **Domain**            | Opens the URL of your store's domain, which is the same as the preview.                                                               |
+| **Last commit**       | Displays the most recent commit made to your store's repository, including the date, month, year, and the person who made the commit. |
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:39]:
```suggestion
| **Last commit**       | Displays the most recent commit to your store's repository, including the date, month, year, and the GitHub user who made the commit. |
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 2]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+| **Status**            | Displays the status of the deployment.                                                                                                |
+
+The **Status** field can have one of the following values:
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:42]: We've just added a new status `Skipped`. Can you add it here pls?

More details on this PR: https://github.com/vtex-sites/faststore.admin/pull/41

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 3]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+| **Commit**                    | Displays the commit hash associated with a particular commit.                                                                                     |
+| **Timestamp**                 | Displays the date and time of the commit in the format `dd/mm/yyyy`.                                                     |
+| **Elapsed time**              | Displays the time it took to complete the commit process.   |
+| **Deploy**                    | Indicates the status of the deployment. Possible statuses are `Success` and `Error`. For more information on the `Error` status, go to the repository and check what failed in the commit.                   |
+| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are `Success` and `Error`. For more information on the `Error` status, go to the repository and check what failed in the commit. *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass.                     |
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:63]: We've just added a new status `Skipped`. Can you add it here pls?

More details on this PR: https://github.com/vtex-sites/faststore.admin/pull/41

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 4]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+| **More actions menu** (`⋮`)    | Opens a card with the following options after being clicked: <li>**Preview**: Opens the store preview for the commit in a new tab.</li>  <li>**Commit**: Opens a specific commit page in GitHub in a new tab.</li>                   |
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:64]:
```suggestion
| **More actions menu** (`⋮`)    | Opens a list with the following options after being clicked: <li>**Preview**: Opens the store preview for the commit in a new tab.</li>  <li>**Commit**: Opens a specific commit page in GitHub in a new tab.</li>                   |
```

File: apps/site/pages/docs/getting-started-glossary.mdx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  The nature of pages created via the [VTEX Headless CMS](/docs/headless-cms-overview). 
+  A Landing Page, a Product Listing Page (PLP), and a Product Detail Page (PDP) are a few examples of different Content Types.
+
+---
+
+## D 
+
+### Design tokens
+Design elements that define the behavior and appearance of components in a store. 
+There are two types: global tokens, which control system-wide interactions, and local tokens, which customize individual components.
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started-glossary.mdx:30]: Should we link to global tokens? https://www.faststore.dev/docs/global-tokens/overview

File: apps/site/pages/docs/getting-started-glossary.mdx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+  ### Override
+  A method to modify a component's props or replacing a native component with a customized version to achieve desired behavior or functionality.
+
+---
+
+  ## S
+
+  ### Sections
+  React page components that wrap up other components, enabling you to create cohesive and functional content for your store, such as the [Hero](/components/organisms/hero) section
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started-glossary.mdx:64]:
```suggestion
  React page components that wrap up other components, enabling you to create cohesive and functional content for your store, such as the [Hero](/components/organisms/hero) section.
```

File: apps/site/pages/docs/getting-started-glossary.mdx (hunk 5 above)
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started-glossary.mdx:30]:
```suggestion
There are two types: [global tokens](https://www.faststore.dev/docs/global-tokens/overview), which control system-wide interactions, and [local tokens](/docs/theming-overview#global-and-local-tokens), which customize individual components. 
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 7]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+
+| Field                         | Description                                                                                                                  |
+| :---------------------------- | :--------------------------------------------------------------------------------------------------------------------------- |
+| **Preview URL**               | Displays a store preview for a specific commit.                                                                                          |
+| **Commit**                    | Displays the commit hash associated with a particular commit.                                                                                     |
+| **Timestamp**                 | Displays the date and time of the commit in the format `dd/mm/yyyy`.                                                     |
+| **Elapsed time**              | Displays the time it took to complete the commit process.   |
+| **Deploy**                    | Indicates the status of the deployment. Possible statuses are `Success` and `Error`. For more information on the `Error` status, go to the repository and check what failed in the commit.                   |
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:62]: Can the [`Skipped` status](https://github.com/vtex-sites/faststore.admin/pull/41) also be an option for the `Deploy` section?

cc: @icazevedo @hellofanny @Gmantiqueira

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 8]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+- `Success`: The deployment was successful. If you want to check the commit associated with it, go to the **Deploys** tab.
+
+- `Error`: There was an error in the pipeline that caused the deployment to fail, but the build can still be successful. 
+To understand what happened, go to the **Deploys** tab, check the last commit, and open the repository for further investigation.
+
+--- 
+
+## Deploys
+
+The **Deploys** view provides a summary of each deployment made in your store's repository.
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:52]:
```suggestion
The **Deploys** section provides a summary of each deployment made in your store's repository.
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 9]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+
+![deploy-view](https://vtexhelp.vtexassets.com/assets/docs/src/dashboard-view-deploys___e5f04f3e5613d52e4b2864ef48134608.png)
+
+| Field                         | Description                                                                                                                  |
+| :---------------------------- | :--------------------------------------------------------------------------------------------------------------------------- |
+| **Preview URL**               | Displays a store preview for a specific commit.                                                                                          |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:58]:
```suggestion
| **Preview URL**               | Displays the URL to the deploy preview of a specific commit.                                                                                          |
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 10]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+| **Elapsed time**              | Displays the time it took to complete the commit process.   |
+| **Deploy**                    | Indicates the status of the deployment. Possible statuses are `Success` and `Failed`. For more information on the `Failed` status, go to the repository and check what failed in the commit.                   |
+| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are <li>`Success` : All checks have passed.</li> <li>`Skipped`:  In the CI process, specific checks were not performed for certain commits This is often done when the commit being tested was never the HEAD commit of the main branch, saving time and resources without compromising code quality.</li> <li>`Failed` : For more information on the `Failed` status, go to the repository and check what failed in the commit.</li>  *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass.                     |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:63]:
```suggestion
| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are <ul><li>`Success` : All checks have passed.</li> <li>`Skipped`:  In the CI process, specific checks were not performed for certain commits. This often occurs when the commit being tested was never the HEAD commit of the main branch, saving time and resources without compromising code quality.</li> <li>`Failed` : For more information on the `Failed` status, go to the repository and check what failed in the commit.</li></ul>  *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the Lighthouse checks to pass.                     |
```

Comment (carolinamenezes) [reply apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:63]: "For more information on the `Failed` status, go to the repository and check what failed in the commit" - Teria que ver o que falhou na audicão do Lighthouse, não? Depois a pessoa vai investigar e corrigir o código dela

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 11]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+| **More actions menu** (`⋮`)    | Opens a list with the following options after being clicked: <li>**Preview**: Opens the store preview for the commit in a new tab.</li>  <li>**Commit**: Opens a specific commit page in GitHub in a new tab.</li>                   |
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:64]:
```suggestion
| **More actions menu** (`⋮`)    | Opens a list with the following options after being clicked: <li>**Preview**: Opens the store preview for the commit in a new tab.</li>  <li>**Commit**: Opens the commit page in GitHub in a new tab.</li>                   |
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 12]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+
+
+
+
+## Deleting the project
+
+To delete your FastStore project, click the **More actions menu** (`⋮`) and **Delete Project**.
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:71]:
```suggestion
To delete your FastStore project, click the **More actions menu** (`⋮`) button. Then, click **Delete Project**.
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 13]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+
+A modal to confirm your choice will open.
+
 <Callout type="warning" emoji="⚠️">
-  This documentation is currently under development.
-</Callout>
\ No newline at end of file
+ Proceed with caution. Deleting the project is irreversible and will permanently remove all deployments and domains associated with it.
+</Callout>
```
Comment (carolinamenezes) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx]: Todo mundo tem acesso a essa botão de deletar ou só usuários com uma role específica?
Seria bom especificar

**Review by carolinamenezes — COMMENTED** [review]
💬 
A doc de Glossary está com a indentação errada. Seria bom arrumar o arquivo dps

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx (hunk 7 above)
Comment (hellofanny) [reply apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx]: I believe so!

File: apps/site/pages/docs/getting-started-glossary.mdx [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+  In other words, an SSG automates the task of coding individual HTML pages and prepares those pages in advance for serving to users.
+
+---
+
+  ## T
+  ### Theming
+  The process of customizing the apperance of a store by using CSS stylesheets known as [themes](/docs/getting-started-glossary#themes). 
+
+  ### Themes
+  Collections of configurable design tokens that define the look-and-feel of the store UI.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started-glossary.mdx:78]:
```suggestion
  Collections of configurable design tokens that define the look and feel of the store's UI
```

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 15]
```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
...
+| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are <li>`Success` : All checks have passed.</li> <li>`Skipped`:  In the CI process, specific checks were not performed for certain commits This is often done when the commit being tested was never the HEAD commit of the main branch, saving time and resources without compromising code quality.</li> <li>`Failed` : For more information on the `Failed` status, go to the repository and check what failed in the commit.</li>  *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass.                     |
+| **More actions menu** (`⋮`)    | Opens a list with the following options after being clicked: <li>**Preview**: Opens the store preview for the commit in a new tab.</li>  <li>**Commit**: Opens the commit page in GitHub in a new tab.</li>                   |
+
+
+
+
+## Deleting the project
+
+To delete your FastStore project, click the **More actions menu** (`⋮`) button. Then, click **Delete Project**.
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:71]:
```suggestion
To delete your FastStore project, click the **More actions menu** (`⋮`). Then, click **Delete Project**.
```

File: apps/site/pages/docs/getting-started-glossary.mdx [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+
+### Override
+A method to modify a component's props or replace a native component with a customized version to achieve the desired behavior or functionality.
+
+---
+
+## S
+
+### Sections
+React components that wrap up other components, facilitating the creation of cohesive and functional content for your storefront. For example, the [Hero](/components/organisms/hero) section. In the context of the [Headless CMS](/docs/headless-cms-overview), sections are predefined and customizable content structures. They act as building blocks that can be populated with content, such as text, images, and media, allowing users to publish content in a more user-friendly and standardized manner. The integration of sections with a CMS streamlines the content creation process and ensures a consistent look and feel across different pages or sections of storefront.
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started-glossary.mdx:64]: I'm wondering where we should link the Hero section, maybe to the section listed [here](https://faststore-site-git-docs-onboarding-dashboard-faststore.vercel.app/docs/reference-guides/list-of-native-sections)?

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 17]
```diff
@@ -11,6 +11,82 @@ import { Callout } from 'nextra-theme-docs'
...
+| **Deploy**                    | Indicates the status of the deployment. Possible statuses are `Success` and `Failed`. For more information on the `Failed` status, go to the repository and check what failed in the commit.                                                                                                                                                                                                                                                                 |
+| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). For more information on the Quality checks values, refer to [Quality checks available values](/docs/getting-started/1-faststore-onboarding/dashboard#quality-checks-available-values). <br></br>*Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass. |
```
Comment (hellofanny) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:66]: It also covers `Integration Tests`, do you think it's worth mentioning it too?

**Review by hellofanny — APPROVED** [review]
LGTM!

File: apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx [hunk 18]
```diff
@@ -11,6 +11,82 @@ import { Callout } from 'nextra-theme-docs'
...
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx:66]:
```suggestion
| **Quality checks**            | Provides an overview of the status of quality checks, including Lighthouse audits and Integration Tests. These checks help you identify and prevent regressions in your store. For more information on the Quality checks values, refer to [Quality checks available values](/docs/getting-started/1-faststore-onboarding/dashboard#quality-checks-available-values). <br></br>*Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass. |
```

**Pull request merged**